lib/base/background/background_worker.dart

```diff
@@ -73,8 +80,9 @@ abstract class BackgroundWorker<BS> {
         response = fn.call(incomingMsg.payload, backgroundState, log);
       } catch (e) {
         exceptionCaught = true;
-        log(LogLevel.WARNING, '$name: backgroundWorkFn threw', ex: e);
-        incomingMsg.sender.send(e);
+        log(LogLevel.WARNING, '$name: backgroundWorkFn threw',
+            exceptionMsg: e.toString());
```

Comment:
This is the main change - now `e.toString()` is send to the main Isolate instead of the `e` exception itself.
Dart supports sending of only simple objects between isolates, and sometimes exceptions are not simple at all - they might contain fields-references-to-methods, stacktraces, etc.
This code crashed the background isolate in cases when the exception was complex.